frontend/src/lib/components/JSSnippet.tsx

```diff
@@ -6,9 +6,9 @@ export function JSSnippet(): JSX.Element {
     const { currentTeam } = useValues(teamLogic)
 
     return (
-        <CodeSnippet language={Language.HTML}>{`<script>
+        <CodeSnippet language={Language.HTML}>{`<script defer>
```

Comment:
it would be easy to lose analytics information like this though, so I'd prefer not having this be the default, and people who know what they're doing can just do this if they want to 🤷‍♂️

Reply:
Seems fair and was the feedback I anticipated, tbh. Happy if an engineer wants to just close this.

Reply:
I'll close, but @Twixes feel free to re-open if you have a differing view

Reply:
(also, kudos for just making the PR, that's awesome to see ❤️ )